packages/Manager/src/main/resources/db/migration/V0019__change_water_alloction_amount_to_numeric.sql

```diff
@@ -0,0 +1,44 @@
+
+UPDATE allocation_amounts SET area_id = 'BoothsSW' WHERE id = 22;
+UPDATE allocation_amounts SET area_id = 'HuangaruaSW' WHERE id = 27;
+UPDATE allocation_amounts SET area_id = 'HuttSW' WHERE id = 32;
+UPDATE allocation_amounts SET area_id = 'KopuarangaSW' WHERE id = 17;
+UPDATE allocation_amounts SET area_id = 'LakeWairarapaSW' WHERE id = 29;
+UPDATE allocation_amounts SET area_id = 'MangaoneSW' WHERE id = 39;
+UPDATE allocation_amounts SET area_id = 'MangatarereSW' WHERE id = 23;
+UPDATE allocation_amounts SET area_id = 'OrongorongoSW' WHERE id = 34;
+UPDATE allocation_amounts SET area_id = 'OtakiSW' WHERE id = 38;
+UPDATE allocation_amounts SET area_id = 'OtukuraSW' WHERE id = 30;
+UPDATE allocation_amounts SET area_id = 'PapawaiSW' WHERE id = 25;
+UPDATE allocation_amounts SET area_id = 'ParkvaleSW' WHERE id = 21;
+UPDATE allocation_amounts SET area_id = 'Ruamahanga_LowerSW' WHERE id = 28;
+UPDATE allocation_amounts SET area_id = 'Ruamahanga_MiddleSW' WHERE id = 26;
+UPDATE allocation_amounts SET area_id = 'Ruamahanga_UpperSW' WHERE id = 20;
+UPDATE allocation_amounts SET area_id = 'TauherenikauSW' WHERE id = 31;
+UPDATE allocation_amounts SET area_id = 'WaikanaeSW' WHERE id = 40;
+UPDATE allocation_amounts SET area_id = 'WaingawaSW' WHERE id = 19;
+UPDATE allocation_amounts SET area_id = 'WainuiomataSW' WHERE id = 33;
+UPDATE allocation_amounts SET area_id = 'WaiohineSW' WHERE id = 24;
+UPDATE allocation_amounts SET area_id = 'WaipouaSW' WHERE id = 18;
+UPDATE allocation_amounts SET area_id = 'WaitohuSW' WHERE id = 37;
+UPDATE allocation_amounts SET area_id = 'Dry RiverGW' WHERE id = 14;
+UPDATE allocation_amounts SET area_id = 'Fernhill TiffenGW' WHERE id = 4;
+UPDATE allocation_amounts SET area_id = 'HuangaruaGW' WHERE id = 12;
+UPDATE allocation_amounts SET area_id = 'LakeGW' WHERE id = 11;
+UPDATE allocation_amounts SET area_id = 'Lower HuttGW' WHERE id = 36;
+UPDATE allocation_amounts SET area_id = 'MangatarereGW' WHERE id = 8;
+UPDATE allocation_amounts SET area_id = 'MartinboroughGW' WHERE id = 13;
+UPDATE allocation_amounts SET area_id = 'OnokeGW' WHERE id = 15;
+UPDATE allocation_amounts SET area_id = 'Parkvale_ConfinedGW' WHERE id = 6;
+UPDATE allocation_amounts SET area_id = 'ParkvaleUnconfinedGW' WHERE id = 7;
+UPDATE allocation_amounts SET area_id = 'RaumatiGW' WHERE id = 41;
+UPDATE allocation_amounts SET area_id = 'TaratahiGW' WHERE id = 5;
+UPDATE allocation_amounts SET area_id = 'TauherenikauGW' WHERE id = 9;
+UPDATE allocation_amounts SET area_id = 'Te HoroGW' WHERE id = 43;
+UPDATE allocation_amounts SET area_id = 'Te Ore OreGW' WHERE id = 1;
+UPDATE allocation_amounts SET area_id = 'Upper HuttGW' WHERE id = 35;
+UPDATE allocation_amounts SET area_id = 'Upper RuamahangaGW' WHERE id = 3;
+UPDATE allocation_amounts SET area_id = 'WaikanaeGW' WHERE id = 42;
+UPDATE allocation_amounts SET area_id = 'WaingawaGW' WHERE id = 2;
+UPDATE allocation_amounts SET area_id = 'WaitohuGW' WHERE id = 44;
```

Comment:
@MartyP233 do you want to quickly spot-check this import based on the spreadsheet you've created?